spec/fixtures/module/declare-global.js

```diff
@@ -0,0 +1 @@
+const global = {}  // eslint-disable-line no-unused-vars
```

Comment:
Instead of having the `eslint` ignore, you could just have `module.exports = global` on the next line and then assert the value returned in the spec was the correct global object expected.